tests/framework/ROM/skl_timedep.xml

```diff
@@ -77,12 +77,14 @@
     <ROM name="knn" subType="SciKitLearn">
       <Features>v0,y0,ang</Features>
       <Target>x,y,time</Target>
+      <pivotParameter>time</pivotParameter>
```

Comment:
This still technically uses the default parameter; can we change to use a different pivot so we really cover the functionality?

Reply:
changed.